python/tvm/contrib/target/vitis_ai.py

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, unused-argument, import-outside-toplevel
+"""Utility to compile Vitis-AI models"""
+
+import os
+
+from tvm.relay.expr import Tuple, Call, TupleGetItem
+import tvm._ffi
+
+import pyxir
+import pyxir.frontend.tvm
+
+
+class CodegenVitisAI:
+    """
+    Traverse Relay expression and convert into PyXIR XGraph format
+    """
+    def __init__(self, model_name, function):
+        self.model_name = model_name
+        self.function = function
+        self.params = {}
+
+    def convert_pyxir(self, target):
```

Comment:
Can we rely on relay's annotation and partitioning, and then offload the partitioned graph to pyxir?

Reply:
Currently, DPU supports  one subgraph per model . For more details refer [here](https://github.com/apache/incubator-tvm/pull/6343#discussion_r479179663).